dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/fat/FATSuite.java

```diff
@@ -28,9 +31,15 @@
 	AcmeConfigVariationsTest.class,
 	AcmeURIConfigVariationsTest.class,
 	AcmeRevocationTest.class
-
-	 })
+})
 public class FATSuite {
+    /*
+     * Run only EE9 tests in LITE mode and run all tests in FULL mode.
+     */
+    @ClassRule
+    public static RepeatTests repeat = RepeatTests.with(new EmptyAction().fullFATOnly()).andWith(new JakartaEE9Action());
```

Comment:
since you only have 1 test enabled for EE9 and you have things configured for full fat, should we always run the EmptyAction() instead of doing it only in full fat?

Reply:
I modified this to run with the EmptyAction in Lite mode as well, since the runtime with it enabled is only ~14-15 mins.